client/src/components/UpcomingInterviews.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
```

Comment:
There's a lot of overlap between this component and `PastInterview`. It would be a good idea to abstract some of the commonalities out into separate reusable components